cas-security-spring-boot-sample/src/main/java/com/kakawait/sample/CasSecuritySpringBootSampleApplication.java

```diff
@@ -136,7 +136,7 @@ public void configure(HttpSecurity http) throws Exception {
     static class ApiSecurityConfiguration extends WebSecurityConfigurerAdapter {
         @Override
         protected void configure(HttpSecurity http) throws Exception {
-            http.antMatcher("/api/**").authorizeRequests().anyRequest().authenticated();
+            http.antMatcher("/api/**").authorizeRequests() /* TODO anyRequest().authenticated()*/;
```

Comment:
Any reason for this todo?

Reply:
Well spotted!

I forgot to do the TODO :-(

I have now solved this by adding an @Order on ApiSecurityConfiguration
and preventing that an antMatchers method is called after the .anyRequest().authenticated() in other configure methods.